geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HScanExecutor.java

```diff
@@ -101,20 +105,24 @@ public RedisResponse executeCommand(Command command,
         return RedisResponse.error(ERROR_SYNTAX);
       }
     }
-
     try {
       matchPattern = convertGlobToRegex(globPattern);
     } catch (PatternSyntaxException e) {
+
       LogService.getLogger().warn(
-          "Could not compile the pattern: '{}' due to the following exception: '{}'. HSCAN will return an empty list.",
+          "Could not compile the pattern: '{}'"
```

Comment:
please change back this formatting

Reply:
hmm-  on my machine I don't see this change.  if it comes back let me know and I'll revert it